content/applications/websites/livechat/overview/get_started.rst

```diff
@@ -2,82 +2,200 @@
+button which contains a link to join directly.
+
+.. image:: get_started/live-chat-widget-code.png
    :align: center
-   :alt: View of the widget tab for Odoo Live Chat
+   :alt: View of the widget tab for Odoo Live Chat.
+
+Participate in live chat
+========================
+
+As explained above, *operators* are the users who will respond to live chat requests from
+customers. The information below outlines the necessary steps for operators participating in
+live chat conversations on an *Odoo* database.
 
-Managing chat requests
-======================
+Set an online chat name
+-----------------------
 
-Conversations initiated by visitors pop up as a direct message, and are shown in *Discuss*.
-Therefore, inquiries can be answered wherever you are in Odoo.
+Before participating in a live chat, operators should update their :guilabel:`Online Chat Name`.
+This is the name that will be displayed to site visitors in the live chat conversation.
 
-.. image:: get_started/discuss.png
+To update the :guilabel:`Online Chat Name`, click on the user name in the upper right corner of any
+page in the database. Select :guilabel:`My Profile`. Scroll to the :guilabel:`Online Chat Name`
+field and enter the preferred name.
+
+.. image:: get_started/live-chat-my-profile.png
    :align: center
-   :alt: View of the discuss application with a message sent through live chat in Odoo
+   :alt: View of the My Profile option in Odoo.
 
-.. note::
-   Conversations are dispatched based on the current workload of the online operators.
+If an :guilabel:`Online Chat Name` is not set, the name displayed will default to the
+:guilabel:`User Name`.
 
-Leave or join a channel
-=======================
+.. example::
+   A user has their full name as their :guilabel:`User Name`, but they do not want to include their
+   last name when participating in live chat. They would then set their :guilabel:`Online Chat
+   Name` to include only their first name.
+
+  .. image:: get_started/live-chat-online-chat-name.png
+     :align: center
+     :alt: View of user profile in Odoo, emphasizing the Online Chat name field.
+
+Join or leave a channel
+-----------------------
+
+To join a live chat channel, go to the :menuselection:`Live Chat` app and click the
+:guilabel:`JOIN` button on the kanban card for the appropriate channel. Any channel where the user
+is currently active will show a :guilabel:`LEAVE` button. Click this button to disconnect from the
+channel.
+
+.. image:: get_started/live-chat-leave-channel.png
+   :align: center
+   :alt: View of a channel form and the option to join a channel for Odoo Live Chat.
+
+.. important::
+   Operators that do not show any activity In Odoo for more than thirty minutes will be considered
+   as disconnected.
+
+Manage live chat requests
+-------------------------
+
+Conversations initiated by site visitors appear as direct messages in the :guilabel:`Discuss`
+application. New conversations will appear in bold.
```

Comment:
I think it would be beneficial to the overall doc if you took the time to expand on this section. Talk about how to navigate to the Discuss app - briefly touch on the Discuss interface/dashboard - where to look for specific Live Chat communications - etc.